addons/web_editor/static/src/js/editor/odoo-editor/src/commands/commands.js

```diff
@@ -196,7 +197,11 @@ export const editorCommands = {
         if (startNode.nodeType === Node.ELEMENT_NODE) {
             if (selection.anchorOffset === 0) {
                 const textNode = editor.document.createTextNode('');
-                startNode.prepend(textNode);
+                if (isVisibleEmpty(startNode)) {
+                    startNode.parentElement.prepend(textNode);
```

Comment:
Don't you mean `startNode.parentNode.insertBefore(textNode, startNode)` ?

Reply:
Yes..!